Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/GoogleCloudStorage.yml

```diff
@@ -103,44 +90,33 @@ script:
       - private
       - projectPrivate
       - publicRead
-      required: false
-      secret: false
-    deprecated: false
+    - defaultValue: eu
```

Comment:
In the documentation it seems that the default value is US.
[create_bucket](https://cloud.google.com/python/docs/reference/storage/latest/google.cloud.storage.client.Client#google_cloud_storage_client_Client_create_bucket)

Reply:
yes that was the reason why we added the parameter location to create buckets in other regions as well. We have set the default value for us to "eu".

Reply:
Yes, but if that's the case, something will break for the customers, who until now knew that they were producing buckets in the US, and suddenly it will be produced for them in the EU